src/settings/.local/share/scripts/post-install.sh

```
@@ -25,7 +25,23 @@ if [[ -d "$INSTALL_DIR/.local/share/nvim/lazy" ]]; then
     cd "$INSTALL_DIR" || exit
 fi
 
-# --- B. Restore Git Identity ---
+# --- B. Reset Tinty Plugins ---
+if [[ -d "$INSTALL_DIR/.local/share/tinted-theming/tinty/repos" ]]; then
+    echo "[Post-Install] Resetting Tinty plugins..."
+    cd "$INSTALL_DIR/.local/share/tinted-theming/tinty/repos" || exit
+    for d in */ ; do
+      if [[ -d "$d/.git" || -f "$d/.git" ]]; then
+          pushd "$d" > /dev/null
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)